modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/skin/Utils.java

```diff
@@ -414,9 +440,35 @@ public static String computeClippedText(Font font, String text, double width,
         }
     }
 
-    public static String computeClippedWrappedText(Font font, String text, double width,
-                                            double height, double lineSpacing, OverrunStyle truncationStyle,
-                                            String ellipsisString, TextBoundsType boundsType) {
+    /**
+     * Computes the actual text to be shown in the Labeled with the text wrapping enabled:
+     * unmodified if it fits into available area,
+     * or with the ellipsis string inserted into strategic place(s) if it does not.
```

Comment:
Similar to previous comment. ellipsis string can be referenced similar to textTruncated?

Reply:
oh, you want to use `{@code ellipsisString}` instead of the words "ellipsis string", got it.
I thought it was clear enough, but yes, can be easily changed.